pkg/omf/cli/omf.install_package.fish

```diff
@@ -1,5 +1,12 @@
-function omf.install_package
+# Return success for at least one successful install.
+function omf.install_package 
+  set -l return_code $OMF_UNKNOWN_ERR
   for search in $argv
     omf.install --pkg $search
+    set -l install_code $status
+    if test $install_code -eq 0
```

Comment:
So, I'm restarting from your commit. I saw that you made the install command to do only an installation (quite logical).

So I have to create an omf.update similar to the install ?

For now I will continue with two command :omf.install and omf.update.